### DISCUSSION TITLE
Waking server with code

### DISCUSSION THREAD

**Discussion by mattiacerutti** [gekigek99/minecraft-vanilla-server-hibernation#172] [Q&A]
Hello, i would like to know if it's possiible (and how) to simulate a player connecting and so waking up the server with an hypothetic script in order to start the server remotely and not with only minecraft

**Comment by mattiacerutti** [comment]
Never mind, got some help from the discord!

**Comment by gekigek99** [comment]
Ok I'm glad you found a solution!

for completeness I wrote a wiki some time ago, check it out:
- https://github.com/gekware/minecraft-server-hibernation/wiki/Client-info%5Cjoin-request
- https://github.com/gekware/minecraft-server-hibernation/wiki/Server-info-response
